package.json

```diff
@@ -5,10 +5,12 @@
 	"main": "index.js",
 	"scripts": {
 		"format": "prettier --write 'packages/**/*.ts'",
+		"api::build": "npm run build --workspace api; npm run build-frontend --workspace api",
```

Comment:
build-frontend can be removed from this script. Probably it came through after the merge conflict resolving